lib/bundle/brew_dumper.rb

```diff
@@ -25,7 +25,9 @@ def dump
         f[:installed_on_request?] || !f[:installed_as_dependency?]
       end
       requested_formula.map do |f|
-        brewline = "brew \"#{f[:full_name]}\""
+        brewline = ""
+        brewline += "# #{f[:desc]}\n" if ARGV.include?("--describe") && f[:desc]
```

Comment:
Do you think it's preferably to display this comment above vs. on the same line to the right of the name? Don't feel strongly but figured it's worth thinking about.

Reply:
Above felt better for reading.

```
# this is some description
brew 'asdf'
# a tool for describing
brew 'sdfgsdfsdfdadfs'
# yet another great tool everyone installs
brew 'wertwert'
```

versus

```
brew 'asdf' # this is some description
brew 'sdfgsdfsdfdadfs' # a tool for describing
brew 'wertwert' # yet another great tool everyone installs
```

The latter is harder to skim by a lot, even when syntax highlighting is present.

Reply:
The only thought reading that is you may want an additional newline after every described line to improve the grouping?

Reply:
I'd thought about that. I played with it a little. I can't decide on _where_ I want to put the newlines. I can insert this into `brew_dumper.rb` on line 35 as of the latest commit and I get reasonable output.

```ruby
brewline += "\n" if ARGV.include?("--describe") && f[:desc]
```

If I omit the `&& f[:desc]`, then _all_ brew lines get an extra newline, regardless of if they have a  description. This looks more consistent.

Honestly, the inconsistency is what makes me not want to have the extra newline.